Feature/#50 tasks definition refactor (#51)
Knotx/knotx-fragments: README.md

failure logic can be adjusted with fallback configuration (no changes in the business logic required).

Knot.x Fragments is designed to build fault-tolerant, back-end integrations such as:
- API Gateways, Backend For Frontend (BFF) for single-page applications (SPA), Web APIs
- documents processing (HTML, JSON, PDF etc) with a templating engine support

## How does it work

Knot.x Fragments is a set of [Handlers](https://github.com/Knotx/knotx-server-http/tree/master/api#routing-handlers)
that are plugged into the [Knot.x Server request processing](https://github.com/Knotx/knotx-server-http#how-does-it-work).

> Processing of Fragments begins by [converting a HTTP request](#supply-fragments) to one or more

Reviewer: ```suggestion
Fragments processing starts with [converting an HTTP request](#supply-fragments) to one or more
```

[Fragments](https://github.com/Knotx/knotx-fragments/tree/master/api#knotx-fragment-api) that are 

> then [evaluated](#evaluate-fragments) and eventually [combined into a HTTP response](#assemble-fragments).

Reviewer: ```suggestion
then [evaluated](#evaluate-fragments) and eventually [combined into an HTTP response](#assemble-fragments).
```
